src/components/PageButtons.vue

```diff
@@ -0,0 +1,71 @@
+<script setup>
+import IconHome from './icons/IconHome.vue';
+
+const props = defineProps({ buttons: { type: Array, required: true } })
+
+
+</script>
+
+<template>
+  <div class="container">
+    <button
+      v-for="(button, index) in buttons"
+      :key="index"
+      class="page"
+    >
+      <aside class="icon-wrapper">
+        <component :is="button.comp" />
+      </aside>
+
+      <span class="page-text">
+        {{ button.name }}
+      </span>
+    </button>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+
+.container {
+    display: flex;
+    flex-direction: column;
+}
+
+button {
+    background-color: transparent;
+    color: #ffff;
+    opacity: 60%;    
+    cursor: pointer;
+    border: none;
+    
+  &:hover {
+    opacity: 100%;
+  }
+}
+
+.page {
+    padding-top: 16px;
+    padding-bottom: 16px;
```

Comment:
Use css short hands instead of declaring different part of various css items.

For this (and since you have padding-left on line 54: `padding: 16px 0 16px 16px;`

Or, with sass:

```
padding: {
  top: 16px;
  bottom: 16px;
  left: 16px;
)
```

Especially try to keep attributes together like having top/bottom here and left a few lines below.